stroke/bids_workflow.py

```diff
@@ -20,8 +20,8 @@ def __init__(self, workflow_element_names: list = None, *args, **kwargs):
             List of the names for the elements of the workflow. Included to be consistent with RAMP API.
 
         """
-        if workflow_element_names is None:
-            workflow_element_names = ["estimator.py", "bids_loader.py"]
+        if(workflow_element_names is None):
```

Comment:
you're such a C programmer ! :)

Reply:
I'm still getting over Python's abuse of whitespace.